flink-connectors/flink-connector-elasticsearch-base/src/test/java/org/apache/flink/connector/elasticsearch/table/ElasticsearchDynamicSinkFactoryBaseTest.java

```diff
@@ -34,6 +37,10 @@
 import java.util.Arrays;
 import java.util.Collections;
 
+import static org.apache.flink.table.factories.FactoryUtil.SINK_PARALLELISM;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
```

Comment:
Flink has switched over to using assertj for tests. Please use the corresponding assertj assertions instead